tests/update_reference_data.sh

```diff
@@ -154,6 +154,7 @@ tests[udq_pyaction]="flow udq_actionx PYACTION_WCONPROD"
 tests[spe1_foam]="flow spe1_foam SPE1FOAM"
 tests[wsegsicd]="flow wsegsicd TEST_WSEGSICD"
 tests[wsegaicd]="flow wsegaicd BASE_MSW_WSEGAICD"
+tests[wsegaicd]="flow wsegvalv BASE_MSW_WSEGVALV"
```

Comment:
This should **probably** say `"wsegvalv"` in order to match the test name.

Reply:
Thanks.